### ISSUE TITLE
cannot set boolean values false when the field is required

### ISSUE THREAD

**Issue by eyadfareh** [pocketbase/pocketbase#917]
if you set a boolean field to be required, you cannot set its value false (at least using the admin panel)
![image](https://user-images.githubusercontent.com/51833505/199750873-715a5102-e2f5-435a-a31b-f4247ef2f6ca.png)

**Comment by ganigeorgiev** [comment]
This seems to be a common misunderstanding I'll see if I can document it better in the future.

In any case, this was recently discussed in https://github.com/pocketbase/pocketbase/issues/907, https://github.com/pocketbase/pocketbase/issues/852, https://github.com/pocketbase/pocketbase/issues/832, https://github.com/pocketbase/pocketbase/issues/526, https://github.com/pocketbase/pocketbase/issues/604, https://github.com/pocketbase/pocketbase/issues/436 and probably others.

In short, it is working as intended because since PocketBase v0.3 we no longer support null for field values and uses zero-defaults. You can check the table with the supported values in https://github.com/pocketbase/pocketbase/releases/tag/v0.3.0.

In v0.8.0 the "Required" constraint was renamed to "Nonempty" and a tooltip was added to explain what does this mean. You could check the live demo - https://pocketbase.io/demo/.
